### ISSUE TITLE
[external] TeX.SX: can't write md5 file when `-output-directory' is used

### ISSUE THREAD

**Issue by pgf-tikz-bot** [pgf-tikz/pgfplots#298]
*Migrated from SourceForge*
https://sourceforge.net/p/pgfplots/feature-requests/64/
Author: mo-gul
Timestamp: 2017-02-09 01:37:59.844000

Please have a look at
    <http://tex.stackexchange.com/questions/243935>

Hopefully I summarize right, when there seems to be a lack of feature (or maybe it is a bug) that externalization fails when `-output-directory` is used (in combination with `\tikzsetexternalprefix`).

(As discussed via email from 08.02.2017 this is added as low-priority feature request.)

**Comment by matthewelmer-tamu** [comment]
Any update on this? I'm having (annoyingly) intermittent issues with this. Sometimes it figures it out after trying again, but sometimes it doesn't and I have no idea why!

**Comment by hmenke** [comment]
No, the `external` library is unmaintained. Use [`memoize`](https://ctan.org/pkg/memoize) or [`robust-externalize`](https://ctan.org/pkg/robust-externalize) instead.
